### PR TITLE
Adds README.

### PR DESCRIPTION
**Pull request by RubyNova**
This PR deals with the README portion of #45 .

This shouldn't take too long to review, but I'd like someone else to have a look.

### CODE REVIEW

**Comment by capnkenny** [comment]
Seems good for now - do you want some proofreading and/or nitpicks?

**Comment by RubyNova** [comment]
Sure nitpick if you want I wrote it in 10 minutes lol.

File: README.md [hunk 1]
```diff
@@ -25,5 +25,8 @@ Currently there are no binary distributions of the engine as of yet, and we are
 ## Example
 Examples will be placed here when we have created some. In the meantime, we advise asking us directly on our discord (invite URL above and below).
 
+## Contributing
+First off, if you want to contribute, thanks! Tickets that are not being resolved by the development team will be plainly marked as such via `Help Wanted` or something similar to that should it change. We also would greatly appreciate being reached out to via our discord to discuss what you could work on. 
+
 ## questions?
```
Comment (capnkenny) [inline README.md]: nit: Capital Q makes it consistent with the rest of the readme

**Comment by tannergooding** [comment]
nit: Merge https://github.com/novelrt/NovelRT/pull/60 first and then add a link to the CoC

**Comment by tannergooding** [comment]
nit: Also wait for https://github.com/novelrt/NovelRT/pull/62 to be merged and add short blurb on the license to the readme.

**Comment by RubyNova** [comment]
poking the bot

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Example
+Examples will be placed here when we have created some. In the meantime, we advise asking us directly on our discord (invite URL above and below).
+
+## Contributing
+First off, if you want to contribute, thanks! Tickets that are not being resolved by the development team will be plainly marked as such via `Help Wanted` or something similar to that should it change. We also would greatly appreciate being reached out to via our discord to discuss what you could work on. 
+
+**NovelRT and everyone contributing (this includes issues, pull requests, the wiki, Discord in all forms, etc.) must abide by our code of conduct, which can be found [here](CODE_OF_CONDUCT.md). Instances of abuse, harrassment or otherwise unacceptable behaviour must either be reported to a NovelRT Discord via direct message or by contacting us via email at admin@novelrt.dev.**
+
+##Licence
```
Comment (tannergooding) [inline README.md]:
```suggestion
## License
```

Comment (RubyNova) [reply README.md]: That is the US spelling.

Comment (tannergooding) [reply README.md]: (or keep the UK spelling, but you need the space 😄)

Comment (RubyNova) [reply README.md]: oooof I can't believe I missed that LOL, thanks

**Pull request merged**